be/src/cloud/cloud_tablet.cpp

```diff
@@ -409,6 +410,10 @@ Result<std::unique_ptr<RowsetWriter>> CloudTablet::create_transient_rowset_write
 }
 
 int64_t CloudTablet::get_cloud_base_compaction_score() const {
+    if (_tablet_meta->compaction_policy() == CUMULATIVE_TIME_SERIES_POLICY) {
```

Comment:
It should be checked whether there is a rowset containing the delete predicate. If so, return the score for base compaction.